api/_mixins/GlobalEventHandlers__Document.json

```diff
@@ -0,0 +1,2821 @@
+        }
+      },
+      "blur_event": {
+        "__compat": {
+          "description": "<code>blur</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/blur_event",
+          "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#handler-onblur",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "9"
+            },
+            "firefox_android": {
+              "version_added": "9"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "≤12.1"
+            },
+            "opera_android": {
+              "version_added": "≤12.1"
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": "1"
+            },
+            "samsunginternet_android": {
+              "version_added": "1.0"
+            },
+            "webview_android": {
+              "version_added": "1"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "cancel_event": {
+        "__compat": {
+          "description": "<code>cancel</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/cancel_event",
+          "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#handler-oncancel",
+          "support": {
+            "chrome": {
+              "version_added": "32"
+            },
+            "chrome_android": {
+              "version_added": "32"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "19"
+            },
+            "opera_android": {
+              "version_added": "19"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": "2.0"
+            },
+            "webview_android": {
+              "version_added": "4.4.3"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "canplay_event": {
```

Comment:
Right now we have:
https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/oncanplay
https://developer.mozilla.org/en-US/docs/web/api/htmlmediaelement/canplay_event

Per IDL, it would be possible to have the event on:
Window
Document
HTMLElement
MathMLElement
SVGElement

I wonder if we wouldn't be better off if we have the events under GlobalEventHandlers in BCD. On MDN we also place them under GlobalEventHandlers and document where they are available (list above) and where they are typically used (in this case HTMLMediaElements like video and audio). What do you think?